src/util.ts

```diff
@@ -0,0 +1,9 @@
+export const pull =
```

Comment:
https://en.wikipedia.org/wiki/Pullback#Precomposition for the name.

This is a bit silly. I'd just rather transform the `subscribe` function, than the arguments being passed to it.